server/routes/charts.js

```diff
@@ -114,9 +114,8 @@ router
       const { dataDb } = req.app.locals
       const chartList = await dataDb
         .collection(collections.charts)
-        .find({ owner: req.user })
+        .find({ $or: [{ owner: req.user }, { sharedWith: req.user }] })
```

Comment:
re: `{ sharedWith: req.user }`, will this check if the `req.user` is in the array of `sharedWith` values or if the `sharedWith` values equals `req.user`?

Reply:
yes to the former, this will check if the req.user value exists in the array of sharedWith